reddit/producer.py

```diff
@@ -0,0 +1,16 @@
+import pika
+import json
+import os
+
+def publish_stock(stocks_found):
+    # Sends the found stocks to the queue  
+    connection = pika.BlockingConnection(
+    pika.ConnectionParameters(host=os.environ.get('REDDIT_RABBITMQ_HOST')))
+    channel = connection.channel()
+
+    channel.queue_declare(queue='scraped_data')
```

Comment:
Ideally, we don't want to have our services create queues, and instead only check if they exist. I believe the way to do this is with setting the `passive` parameter of this method to `True`

Reply:
I will add a TODO